configure.ac

```diff
@@ -278,6 +279,8 @@ fi
 OPAL_SET_LIB_PREFIX([])
 m4_ifdef([project_orte],
           [ORTE_SET_LIB_PREFIX([])])
+m4_ifdef([project_ompi],
+          [OMPI_SET_LIB_PREFIX([ompi_lib_prefix])])
```

Comment:
I'm a little confused -- I see that we have the mechanisms here to add prefixes for the OPAL and ORTE libraries, but those don't appear to be *used* anywhere (i.e., we just pass in blank strings, so the prefixes are always empty -- there's no way to *not* have them be empty).

I'm therefore trying to remember why this functionality is here...?  😕